src/main/resources/db/migration/V1_79__update_procedure_payment_files_to_clob_write_to_clob.sql

```diff
@@ -0,0 +1,97 @@
+                coalesce(REPLACE(REPLACE(REPLACE(pd.bank_ac_name, '|', ' '), chr(10), ' '), chr(13),' '), '') bank_ac_name,
+                coalesce(REPLACE(REPLACE(REPLACE(pd.bank_ac_number, '|', ' '), chr(10), ' '), chr(13),' '), '') bank_ac_number,
+                coalesce(REPLACE(REPLACE(REPLACE(pd.build_soc_number, '|', ' '), chr(10), ' '), chr(13),' '), '') build_soc_number,
+                coalesce(REPLACE(REPLACE(REPLACE(pd.address_line_1, '|', ' '), chr(10), ' '), chr(13),' '), '') address_line1,
+                coalesce(REPLACE(REPLACE(REPLACE(pd.address_line_2, '|', ' '), chr(10), ' '), chr(13),' '), '') address_line2,
+                coalesce(REPLACE(REPLACE(REPLACE(pd.address_line_3, '|', ' '), chr(10), ' '), chr(13),' '), '') address_line3,
+                coalesce(REPLACE(REPLACE(REPLACE(pd.address_line_4, '|', ' '), chr(10), ' '), chr(13),' '), '') address_line4,
+                coalesce(REPLACE(REPLACE(REPLACE(pd.address_line_5, '|', ' '), chr(10), ' '), chr(13),' '), '') address_line5,
+                coalesce(REPLACE(REPLACE(REPLACE(pd.postcode, '|', ' '), chr(10), ' '), chr(13), ' '), '') postcode,
+                pd.auth_code,
+                coalesce(REPLACE(REPLACE(REPLACE(pd.juror_name, '|', ' '), chr(10), ' '), chr(13), ' '), '') name,
+                pd.loc_cost_centre,
+                pd.travel_total,
+                pd.subsistence_total as sub_total,
+                pd.financial_loss_total as floss_total,
+                pd.creation_date as sub_date
+         FROM juror_mod.payment_data pd
+         WHERE date_trunc('day', pd.creation_date) = p_creation_date
+
+    LOOP
+	    out_rec :=  c_extract.loc_code || c_extract.unique_id || '|' || to_char(c_extract.creation_date, 'DD-Mon-YYYY') || '|'
+	    			|| lpad(to_char(c_extract.expense_total, '9999990.90'), 11) || '|' || rpad(c_extract.loc_code || c_extract.part_invoice, 50) || '|'
+	    			|| to_char(c_extract.creation_date, 'DD-Mon-YYYY') || '|' || c_extract.bank_sort_code || '|'
+	    			|| rpad(c_extract.bank_ac_name, 18) || '|' || rpad(c_extract.bank_ac_number, 8) || '|' || rpad(c_extract.build_soc_number, 18);
+
+	 	out_rec2 :=  '|' || rpad(c_extract.address_line1, 35) || '|'
+	 				|| rpad(c_extract.address_line2, 35) || '|'
+	  				|| rpad(c_extract.address_line3, 35) || '|'
+	  				|| rpad(c_extract.address_line4, 35);
+
+        IF c_extract.travel_total IS NOT NULL then
+            out_rec3 := '|' || rpad(c_extract.address_line5, 35) || '|'
+            			|| rpad(c_extract.postcode, 20) || '|'
+            			|| c_extract.auth_code || '|'
+            			|| rpad(c_extract.name, 50) || '|'
+            			|| c_extract.loc_cost_centre || '|' || '2' || '|'
+                        || lpad(to_char(c_extract.travel_total, '9999990.90'), 11) || '|'
+                   		|| to_char(c_extract.sub_date, 'DD-Mon-YYYY');
+           c_lob := concat(c_lob, out_rec, out_rec2, out_rec3, chr(10));
```

Comment:
do we need to be careful here if any of the out_rec values are null - should we be throwing an exception or concatenating a padded blank string to maintain the expected fixed width?

if we use concat and treat null as an empty string it might cause problems downstream reading the fixed width dataset where the character positions may be out of line 🤔

Reply:
Discussed the change with BAU and confirmed above is okay.   Example of how data is generated with/without bank details: 
<img width="1192" alt="image" src="https://github.com/hmcts/juror-api/assets/148471056/8e312914-37d9-4b42-8388-e3d00bbff27c">

Reply:
yeah I was happy with the bank details changes ... it was where we concat each out_rec on line 73 - I was concerned if out_rec2 was null for example, we still need to pad for out_rec2's length before appending out_rec3 don't we?